src/find_zero.jl

```diff
@@ -586,17 +586,16 @@ julia> find_zero((sin, cos, x->-sin(x)), 3.0, Roots.Halley())  # use Halley's me
 
 Changing tolerances.
 
-```
+```jldoctest find_zero
 julia> fn = x -> (2x*cos(x) + x^2 - 3)^10/(x^2 + 1);
 
 julia> x0, xstar = 3.0,  2.9947567209477;
 
 julia> find_zero(fn, x0, Order2()) ≈ xstar
-true
+false
```

Comment:
This one is odd too, as locally it works for me:

```
julia> fn = x -> (2x*cos(x) + x^2 - 3)^10/(x^2 + 1);

julia> x0, xstar = 3.0,  2.9947567209477;

julia> find_zero(fn, x0, Order2()) ≈ xstar
true

julia> find_zero(fn, x0, Order2()), xstar
(2.9947567209477, 2.9947567209477)
```

Reply:
In my environment, it doesn't work. Had you tried in `1.6.0`?

Reply:
done